src/python/sdk/examples/trade.py

```diff
@@ -0,0 +1,372 @@
+    print(f"Econia address (from view function): {get_address(viewer)}")
+
+    market_id = setup_market(faucet_client, viewer)
+
+    account_A = setup_new_account(viewer, faucet_client, market_id)
+    print(f"Account A was set-up: {account_A.account_address}")
+    # Bid to purchase 1 whole ETH at a price of 1 whole USDC per lot!
+    # = $1000/ETH since there are 1000 lots in a whole ETH & 1 tick = 0.001 USDC
+    buy_base_lots = 1 * (10**3)
+    buy_ticks_per_lot = 1 * (10**3)
+    place_limit_order(Side.BID, account_A, market_id, buy_base_lots, buy_ticks_per_lot)
+    events = get_place_limit_order_events(viewer, account_A.account_address, market_id, 0)
+    print("")
+    report_place_limit_order_event(
+        list(filter(lambda ev: ev["data"]["side"] == Side.BID, events))[0]
+    )
+    # Ask to sell 1 whole ETH at a price of 2 whole USDC per lot!
+    # = $2000/ETH since there are 1000 lots in a whole ETH & 1 tick = 0.001 USDC
+    sell_base_lots = 1 * (10**3)
+    sell_ticks_per_lot = 2 * (10**3)
+    place_limit_order(Side.ASK, account_A, market_id, sell_base_lots, sell_ticks_per_lot)
+    events = get_place_limit_order_events(viewer, account_A.account_address, market_id, 0)
+    print("")
+    report_place_limit_order_event(
+        list(filter(lambda ev: ev["data"]["side"] == Side.ASK, events))[0]
+    )
+
+    print(f"Account A has placing limit orders.")
+
+    report_best_price_levels(viewer, market_id)
+    
+    account_B = setup_new_account(viewer, faucet_client, market_id)
+    print(f"Account B was set-up: {account_B.account_address}")
+    place_market_order(Side.BID, account_B, market_id, 500) # Buy 0.5 ETH
+    place_market_order(Side.ASK, account_B, market_id, 500) # Sell 0.5 ETH
+    print(f"Account B has finished taking!")
+
+    report_best_price_levels(viewer, market_id)
+
+    client_A = EconiaClient(NODE_URL, ECONIA_ADDR, account_A)
+    calldata1 = cancel_all_orders_user(ECONIA_ADDR, market_id, Side.ASK)
+    client_A.submit_tx_wait(calldata1)
+    calldata2 = cancel_all_orders_user(ECONIA_ADDR, market_id, Side.BID)
+    client_A.submit_tx_wait(calldata2)
+    print(f"\nAccount A has cancelled all of their orders!")
+
+    report_best_price_levels(viewer, market_id)
+
+    place_limit_orders_at_market(viewer, account_A, market_id, 100, buy_ticks_per_lot, sell_ticks_per_lot)
+    place_limit_orders_at_market(viewer, account_A, market_id, 200, buy_ticks_per_lot, sell_ticks_per_lot)
+    place_limit_orders_at_market(viewer, account_A, market_id, 300, buy_ticks_per_lot, sell_ticks_per_lot)
+    place_limit_orders_at_market(viewer, account_A, market_id, 400, buy_ticks_per_lot, sell_ticks_per_lot)
+    place_limit_orders_at_market(viewer, account_A, market_id, 500, buy_ticks_per_lot, sell_ticks_per_lot)
+    print("\nAccount A has created multiple competitive limit orders!")
+    
+
+    report_best_price_levels(viewer, market_id)
+
+    volume = 100 + 200 + 300 + 400 + 500
```

Comment:
What is this variable for?

Reply:
it was to-be-used, was in the middle of something with this commit